### PR TITLE
Update `dagbag_size` documentation

### LINKED ISSUE #10431: Metrics Documentation, dag_bag_size
In the airflow documentation metrics section there is a field called dag_bag_size and the description is just "DAG bag size", this is fairly unhelpful for trying to gain insight into the metrics, so would it be possible for this description to either be improved, or linked to a more accurate description?

In the docstring, the description of a dag bag is "a collection of dags, parsed out of a folder tree and has high level configuration settings", but it's unclear which dags are actually in this collection during runtime. For example, is this every dag displayed on the webserver UI, or is it just the dags that are turned on? (I'm fairly certain from monitoring that it's the latter)

### PR DESCRIPTION
**Pull request by javatarz**
closes: #10431

### CODE REVIEW

**Review by kaxil — APPROVED** [review]

**Comment by potiuk** [comment]
There is no need to set the kable if it has been approved/merged:

A pull request is considered approved once it has an overall approving review from maintainers, or has been merged by maintainers, or has been given the 'hacktoberfest-accepted' label.

**Pull request merged**
